examples/rpc-db/Cargo.toml

```diff
@@ -11,7 +11,7 @@ jsonrpsee.workspace = true
 reth.workspace = true
 reth-chainspec.workspace = true
 reth-db.workspace = true
-reth-db-api.workspace = true
+reth-db-models.workspace = true
```

Comment:
can we reuse this from reth-db?
since ClientVersion is part of reth-db API it should be exported from there